samples/wifi/sta/README.rst

```diff
@@ -23,6 +23,13 @@ This sample can perform Wi-Fi operations such as connect and disconnect in the 2
 
 Using this sample, the development kit can connect to the specified access point in :abbr:`STA (Station)` mode.
 
+QSPI encryption
+***************
+This sample by default enables QSPI encryption by using the factory default key (all F's), but you can change the key by
```

Comment:
"key can be changed"  instead of "you can change the key" might sound better? :)

Reply:
Active voice is preferred hence suggested to keep the earlier version.